uber/models.py

```diff
@@ -1324,6 +1348,22 @@ def past_years_json(self):
         return json.loads(self.past_years or '[]')
 
 
+class WatchList(MagModel):
```

Comment:
Hmm, good to know.  Now that I think about it, you're probably my canary in the coal mine on this one.  Like, she's unlikely to murder me BEFORE she murders you, so I'll just keep an eye on police reports of homicide in your area to gauge how nitpicky I can be on these code reviews.